up42/main.py

```diff
@@ -51,7 +89,7 @@ def check_auth(func):
 
     @functools.wraps(func)  # required for mkdocstrings
     def inner(*args, **kwargs):
-        if _auth is None:
+        if workspace.auth is None:
             raise RuntimeError("Not authenticated, call up42.authenticate() first")
```

Comment:
This part is not needed anymore, since it will throw error when calling auth anyway